### PR TITLE
Add schema directory work item

### PR DESCRIPTION
**Pull request by csuwildcat**

### CODE REVIEW

File: work_items/schema_directory.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+The DIF Schema Directory will act as a source of curated references to aid 
+developers in finding schemas that address their needs. When seeking to add 
+to the directory, the group will first seek out existing schemas from external 
+sources that cover the motivating need for the addition. If no schema exists 
+externally that addresses the need, the group may elect to generate and publish 
+new schemas and publish them within the directory project/repo.
+
+## Work Item Owners
+- Daniel Buchner (@github, email@example.com, or both)
```
Comment (rado0x54) [inline work_items/schema_directory.md:11]:
```suggestion
- Daniel Buchner (@github, email@example.com, or both)
- Martin Riedel (@rado0x54)
```

File: work_items/schema_directory.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+- Martin Riedel (@rado0x54)
```
Comment (rado0x54) [inline work_items/schema_directory.md:12]: Would @wyc / spruce be interested to join?

Comment (rado0x54) [reply work_items/schema_directory.md:12]: and @OR13

File: work_items/schema_directory.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+## Deliverables
+- Process for evaluation and inclusion of a new schema category or reference
+- Initial directory populated with a significant enough number of schemas to provide value
+- Publication of the directory on the DIF website
+
+## Meetings
+- Meetings will be held once a month, with async eval of PRs for additions being 
+  the primary mode of collaboration.
+- Meetings will primarily be focused on review of any outstanding issues.
```
Comment (rado0x54) [inline work_items/schema_directory.md:29]: Add prior art section in relation to VC Working Group 2.0, specifically around credentialSchema.

Comment (bumblefudge) [reply work_items/schema_directory.md:29]: https://w3c-ccg.github.io/vc-json-schemas/v2/index.html ?

File: work_items/schema_directory.md (hunk 2 above)
Comment (rado0x54) [reply work_items/schema_directory.md:12]: @csuwildcat Ideally we find another owner.

Comment (bumblefudge) [reply work_items/schema_directory.md:12]: 2 NON-CHAIR codeowners should be more explicit as a req :D

Comment (bumblefudge) [reply work_items/schema_directory.md:13]: welcome aboard, @JackSophtron !

**Pull request merged**